cli/src/main/java/org/infinispan/cli/download/Artifact.java

```diff
@@ -0,0 +1,22 @@
+package org.infinispan.cli.download;
```

Comment:
Nitpick, but wouldn't `org.infinispan.cli.aritfacts` be more appropriate? Not all artfacts require a download.

Reply:
Done